README.md

```diff
@@ -73,9 +74,9 @@ curl
 
 #### Linux
 
-These package names are in the debian / ubuntu style. Feel free to PR your own guide for a different distro.
+These package names are in the Debian / Ubuntu style. Feel free to PR your own guide for a different distro.
 
-Runtime dependencies for **linux** are (debian/ubuntu):
+Runtime dependencies for **Linux** are (Debian/Ubuntu):
 ```
 libz-dev
```

Comment:
we might want to delete this section while we're here - the "other distributions (e.g. Arch)" section is more informative and these package names are outdated